src/onnx/onnx_parser.cpp

```diff
@@ -609,6 +625,13 @@ shape::type_t get_type(int dtype)
     case 11: return shape::double_type;
     case 12: return shape::uint32_type;
     case 13: return shape::uint64_type;
+    case 18: return shape::fp8e4m3fnuz_type;
+    case 14:
+    case 15:
+    case 16:
+    case 17:
+    case 19:
+    case 20:
```

Comment:
Why have these if we don't support those datatypes? We also don't support sparse_tensor_type which is 8.

Reply:
I have only added `E4M3FNUZ` type inside the MIGraphX IR so far. 

That's why only `case 18` is supported.

Reply:
`sparse_tensor_type` is for something else. It is TypeProto something not related to data type.